Demo/Common/Utils/APIClientMock.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+
+typealias MockedResult = Result<Response, Error>
+
+final class APIClientMock: APIClientProtocol {
```

Comment:
I have copied APIClientMock to Demo folder so we can copy Demo folder during CI integration. Origin of APIClientMock is stored in Tests